pkg/tlshelpers/tlshelpers.go

```diff
@@ -135,3 +135,16 @@ func getCommonNameFromOrg(org string) (string, error) {
 	}
 	return "", errors.New("unable to recognise secret name")
 }
+
+// WhitelistEtcdCipherSuites whitelists valid ciphers for use with etcd.
+// TODO move upstream
+func WhitelistEtcdCipherSuites(cipherSuites []string) []string {
+	whitelist := []string{}
+	for _, cipher := range cipherSuites {
+		_, ok := tlsutil.GetCipherSuite(cipher)
```

Comment:
Would it make sense to log failed cipher lookups to aid in troubleshooting?

Reply:
yeah I will add test coverage and logging